hypothesis-python/src/hypothesis/internal/conjecture/data.py

```diff
@@ -982,7 +982,8 @@ def draw_bits(self, n, forced=None):
             index = self.__bytes_drawn
             buf = self.__prefix[index : index + n_bytes]
             # We always draw prefixes as a whole number of blocks
```

Comment:
Comment is now out of date - could add "unless optimising" and `assert ... or current_phase() == Phase.target`, or just delete it.